.husky/pre-commit

```diff
@@ -1,3 +1,2 @@
 yarn lint-staged --concurrent false
 yarn delete-changelog
-git add .
```

Comment:
i replaced this with a more precise add in `dev/delete-changelog.mjs`, as this was causing all unstaged changes to be committed accidentally